client/src/components/DatasetInformation/services.js

```diff
@@ -16,3 +16,18 @@ export async function sendErrorReport(dataset, message, email) {
         rethrowSimple(e);
     }
 }
+
+export async function setAttributes(datasetId, settings, operation) {
+    const payload = {
+        dataset_id: datasetId,
+        operation: operation,
+        ...settings,
+    };
+    const url = `${getAppRoot()}dataset/set_edit`;
```

Comment:
I'll click approve because I know it is important to replace backbone with VueJS for other projects. But I will forever agree with Carl's old arguments that all of this stuff should be designed API first. https://github.com/galaxyproject/galaxy/issues/12332

Reply:
Thank @jmchilton for looking into it and for your help throughout this project. I agree with you.